newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrObjectSelector.js

```diff
@@ -391,7 +391,10 @@ export default class InstructionOrObjectSelector extends React.PureComponent<
                 <Column expand noMargin>
                   <Tabs
                     value={currentTab}
-                    onChange={onChangeTab}
+                    onChange={(e) => {
+                      onChangeTab(e);
+                      this._searchBar.current.focus();}
```

Comment:
Are you sure about this one? This means that when I change from Objects to Other conditions/actions tab, the search bar will "steal the focus". Might not be expected from someone navigating with the keyboard?

Especially because the search is a global search, searching in everything - so users probably don't want to get send back to the search field?